NEWS.txt

```diff
@@ -3,12 +3,82 @@ Release Notes
 =============
 
 
+Theano 0.9.0rc1 (15th of February, 2017)
+========================================
+
+This release extends the 0.9.0beta1 and announces the upcoming final release 0.9.
+
+Highlights (since 0.9.0beta1):
+ - Better integration of Theano+libgpuarray packages into conda distribution
+ - Better handling of Windows end-lines into C codes
+ - Better compatibility with NumPy 1.12
+ - Faster scan optimizations
+ - Fixed broadcast checking in scan
+ - Bug fixes related to merge optimizer
+ - many other bug fixes and improvements
+ - Updated documentation
+
+ - Old GPU back-end:
+   * In MRG, replaced method `multinomial_wo_replacement()` with new method `choice()`
```

Comment:
This is not related to the gpu. This is a user interface. Put this in the section: "Interface changes"